src/scripts/unmapped_docs.py

```diff
@@ -58,6 +58,7 @@ def update_mapping_progress_in_docs():
             'Tot terms': f"{len(df):,}",
             'Tot excluded': f"{len(df[df['is_excluded'] == True]):,}",
             'Tot deprecated': n_deprecated,
+            'Tot deprecated unmapped': f"{len(df[(df['is_deprecated'] == True) & (df['is_mapped'] == False)]):,}",
```

Comment:
Per [#226](https://github.com/monarch-initiative/mondo-ingest/issues/226#issuecomment-1468044153), I added a new column `Tot deprecated unmapped`.

I also double checked the logic to make sure that obsoletes/deprecated terms are not included in the "unmapped mappables". Indeed, it appears they are not:
```python
        mappable_df = df[(df['is_excluded'] == False) & (df['is_deprecated'] == False)]
        unmapped_mappable_df = mappable_df[mappable_df['is_mapped'] == False]
```

Reply:
Very unfortunate. I hoped these large numbers in the unmapped mappable column would somehow decrease..

Reply:
Yeah, as I probably mentioned on our last call, I was not under the impression that this was bugged. All I did here was add a new column. It doesn't seem that there is a need to edit the calculations for 'mappable unmapped', so the number didn't change.

If you want, we can do additional checking. Like we could look through these TSVs / dataframes and just double check a bunch of terms and make sure they are being correctly marked as mapped, etc.